test/io/testccio.py

```diff
@@ -111,6 +111,25 @@ def test_multi_url_io(self):
     #    self.assertIsInstance(self.ccopen(StringIO.StringIO(""), cjson=True), cclib.io.cjsonreader.CJSON)
 
 
+class _determine_output_formatTest(unittest.TestCase):
+
+    def setUp(self):
+        self._determine_output_format = cclib.io.ccio._determine_output_format
+
+    def test_outputclass(self):
+        """Does the function determine output class as expected"""
+        absdir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Changed. Now the test passes the extension as a string.
And there is a test to send an unsupported extension.